airbyte-cdk/python/airbyte_cdk/sources/streams/concurrent/partitions/record.py

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+from typing import Any, Mapping
+
+
+class Record:
+    """
+    Represents a record read from a stream.
+    """
+
+    def __init__(self, data: Mapping[str, Any]):
```

Comment:
@maxi297's comment on [implementation PR](https://github.com/airbytehq/airbyte/pull/30113#discussion_r1317676782)
> A StreamData can be a AirbyteMessage. Are all AirbyteMessage a record?
I updated the constructor to only accept the data mapping. We shouldn't pass in an AirbyteMessage